packages/api/src/features.json

```diff
@@ -0,0 +1 @@
+{"cs_chat_livechat":true,"cs_chat_whatsapp":true,"social_login_fb":false}
```

Comment:
This will be commited to the code base using the maste branch workflow

Reply:
Couple of notes
1. Name of the file doe not tell anything about what is inside. Shall you call it `app-config` ?
2. Is it correctly formatted? Maybe better if it's pretty-printed?

Reply:
Will rename it. For now i just used the name that we configured in Fire Base (https://deriv-static.firebaseio.com/features.json)

Reply:
I see, you get it auto-populated. So, formatting issue is not relevant then.

Reply:
exactly.